### ISSUE TITLE
Contenteditables is not working

### ISSUE THREAD

**Issue by lakhanOpen** [yuku-t/jquery-textcomplete#102]
Hello ,

Hope you are doing great,

Your plugin is great but there is still one issue in your
plugin

your latest js is not working perfectly for ContentEditable example.

So I have tried your old js for my requirement

but it also create some issue.

Example :

in my editable div before searching using @ I have to write any one character then I also have to give space then search is working perfectly.

but if I try directly start with @ then its not working perfectly.

Please check it in ie9 and latest version of firefox.

Please help me

Thanks

**Comment by johandanforth** [comment]
This is still an issue, even the demo of contenteditable on the demo-page isn't working with IE 11 :(

**Comment by jozefvaclavik** [comment]
:+1: having same issue with version 0.5.1 and to give a little more context, it's not that I have to type something before. Simply pressing arrow keys will allow me to fire autocomplete. My match is `/\B@(\w*)$/`.

**Comment by jozefvaclavik** [comment]
If anyone is hitting this problem, we _resolved_ issue I described above by triggering key event after selection
```
$('#awesomeElement').textcomplete([superStrategy]).on(
  {
    'textComplete:select': function (e, value, strategy) {
      $(this).trigger('keyup');
    }
  }
);
```

**Comment by yuku-t** [comment]
I'm not sure the problem has been actually fixed, but I couldn't reproduce it in v1.1.0, so I close the issue. Thanks for the comments though!
